railties/lib/rails/generators/resource_helpers.rb

```diff
@@ -16,12 +16,10 @@ def self.included(base) #:nodoc:
       def initialize(*args) #:nodoc:
         super
         if options[:model_name]
-          controller_name = name
           self.name = options[:model_name]
```

Comment:
Name gets changed here, so we cannot move `controller_name` assignment below this line.

Reply:
Interesting. It appears that what looked like a cosmetic change caused unexpected downstream issues...